src/sidebar/services/test/frame-sync-test.js

```diff
@@ -102,19 +104,41 @@ describe('FrameSyncService', () => {
       .register('store', { value: fakeStore })
       .register('frameSync', FrameSyncService)
       .get('frameSync');
-
-    frameSync.connect();
   });
 
   afterEach(() => {
     $imports.$restore();
   });
 
+  describe('#connect', () => {
+    it('establishes a connection to the parent host/guest frame', () => {
```

Comment:
This is splitting hairs, but it doesn't have any knowledge about the host-guest-iness of the parent frame; it's merely establishing a connection with whatever the parent frame is.